src/torManager/torManager.ts

```diff
@@ -71,22 +72,45 @@ export class Tor {
         const file = fs.readFileSync(this.torPidPath)
         oldTorPid = Number(file.toString())
       }
+      let counter = 0
 
-      if (oldTorPid && process.platform !== 'win32') {
-        child_process.exec(this.torProcessNameCommand(oldTorPid.toString()), (err, stdout, _stderr) => {
-          if (err) {
-            log.error(err)
-          }
-          if (stdout.trim() === 'tor') {
-            process.kill(oldTorPid, 'SIGTERM')
-          } else {
-            fs.unlinkSync(this.torPidPath)
-          }
-          this.spawnTor(resolve)
-        })
-      } else {
-        this.spawnTor(resolve)
+      const foo = async () => {
+        if (counter > repeat) {
+          // eslint-disable-next-line
+          reject()
```

Comment:
Usually reject() takes error as parameter. So sth like
```
reject(new Error(`Failed to spawn tor ${coutner} times`))`
```
maybe if you did that you wouldn't need to skip eslint?